lib/fathead/jquery/article-alias.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var inputFileName = 'articles.txt';
+var outputFileName = 'redirects.txt';
+var filePath = ''
+
+var appendString = ' example,';
+
+var fs = require('fs');
+var readline = require('readline');
+var rl = readline.createInterface(fs.createReadStream(filePath + inputFileName), fs.createWriteStream(filePath + outputFileName,{'flags': 'a'}));
+
+function parseInput(inputString) {
+    var outputString = inputString;
+
+    function upperToLowerSpace(match) {
+        return ' ' + match.toLowerCase();
+    }
+
+    outputString = outputString.replace('jQuery', '').replace(/[\(\)]/g, '').replace(/[\.\:\-]/g, ' ').replace(/([A-Z]+)/g, upperToLowerSpace).replace(/\ \ /g, ' ').trim();
```

Comment:
@Caldwerl This looks great! 👍 

One thing though, can you add another line for each camel case article, besides the split words, with the original title, like this example:
`remove class example,.removeClass()`
`removeClass example,.removeClass()`

Also, committing the redirects.txt will trigger the deploy of the fathead to beta.duckduckgo.com, so we can test it :)